tests/test_events.py

```diff
@@ -68,3 +68,34 @@ def test_event_has_proper_implementation(one_event, another_event):
     # str test
     assert "object at 0x" not in str(one_event), \
         "Event has a proper str method"
+
+@pytest.mark.parametrize("one_event", [
+    UserUttered("/greet", {"intent": "greet", "confidence": 1.0}, []),
+
+    TopicSet("my_topic"),
+
+    SlotSet("my_slot", "value"),
+
+    Restarted(),
+
+    AllSlotsReset(),
+
+    ConversationPaused(),
+
+    ConversationResumed(),
+
+    StoryExported(),
+
+    ActionReverted(),
+
+    ActionExecuted("my_action"),
+
+    BotUttered("my_text", "my_data"),
+
+    ReminderScheduled("my_action", "now")
+])
+def test_serialisation(one_event):
+    story_string = one_event.as_story_string()
+    new_event = Event.from_story_string(story_string, one_event.as_dict(), None)
```

Comment:
@tmbo should this work for all events? (does for reminderscheduled) - do we need a real `domain` for any of these?

Reply:
Yes, I think at least `ActionExecuted` needs it. best is to add `default_domain` as a parameter - will be filled automatically and use that one. The default_domain also has a slot `name` that you can set to a value. Best is to also use an action that is in there, so e.g. `utter_greet`.

Reply:
alright will update that - didn't work for `TopicSet` but that might also be related to domain